### PR TITLE
Added Header,Input & ProgressBar to Homepage

### PR DESCRIPTION
**Pull request by SergeyMOV**

### CODE REVIEW

**Review by actyralf — CHANGES_REQUESTED** [review]
Hi Sergey,
I would suggest that you use styled components instead of classnames in `GlobalStyles`.
For example, instead of having `<button className="MoneyInpbt">Start</button>`you could define a `StyledButton`component like this:
```
const StyledButton=styled.button`
height: 30px;
`
```
and use this instead. Furthermore you tend to give a separate class for every single element - please think of reusing your classes (or styled components). The `.MoneyInput` class or the `.MoneyInpbt` class have nothing special regarding "Money", so these could be generic input or button components.

Please try to develop "Mobile first"and test your code in the mobile device view.

File: components/Input with Start button.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function Entryfield() {
+  return (
+    <>
+      <input className="MoneyInput" min="1" max="9000" type="number" />
+      <button className="MoneyInpbt">Start</button>
+    </>
+  );
```
Comment (actyralf) [inline components/Input with Start button.js]: Please use a filename without blanks - it should reflect the component name too: rename `Input with Start button.js` to `Entryfield.js`

File: styles/GlobalStyles.js [hunk 2]
```diff
@@ -10,6 +10,47 @@ const GlobalStyles = createGlobalStyle`
...
+    left:50%;
+    margin:0;
+    }
+    .Procentage2{
+        transform:rotate(90deg);
+        position:absolute;
+        margin:0;
+        top:47%;
+        left:40%;
+        }
```
Comment (actyralf) [inline styles/GlobalStyles.js]: Please consider to move these classes into StyledComponents.

File: pages/_document.js [hunk 3]
```diff
@@ -3,12 +3,7 @@ import {Html, Head, Main, NextScript} from "next/document";
 export default function Document() {
   return (
     <Html>
-      <Head>
-        <link
-          rel="icon"
-          href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🐙</text></svg>"
-        />
-      </Head>
+      <Head></Head>
```
Comment (stefankorder) [inline pages/_document.js]: You could call the `Head` component as a self closing tag. 
```suggestion
      <Head />
```

**Review by stefankorder — CHANGES_REQUESTED** [review]
Good work. Please take a look at my comment.

File: pages/_document.js (hunk 3 above)
Comment (SergeyMOV) [reply pages/_document.js]: I did not touch the Head Component before. It was already written incorrectly in the Default template.

**Pull request merged**